collect_app/src/test/java/org/odk/collect/android/widgets/VideoWidgetTest.java

```diff
@@ -131,6 +131,10 @@ public void usingReadOnlyOptionShouldMakeAllClickableElementsDisabled() {
 
         assertThat(getWidget().captureButton.getVisibility(), is(View.GONE));
         assertThat(getWidget().chooseButton.getVisibility(), is(View.GONE));
-        assertThat(getWidget().playButton.getVisibility(), is(View.VISIBLE));
+        if (getWidget().getQuestionDetails().getPrompt().getAnswerText() == null) {
```

Comment:
Surely the `else` case of this case is never run right? I think you need a second test that drives out the behaviour when the widget's prompt is read only and the prompt has an answer.

Reply:
Yes